Nos/Views/NotificationCard.swift

```diff
@@ -35,23 +35,25 @@ struct NotificationCard: View {
                 VStack {
                     HStack {
                         Text(viewModel.actionText)
-                            .lineLimit(1)
+                            .lineLimit(2)
                         Spacer()
                     }
-                    HStack {
-                        let contentText = Text("\"" + (content ?? "") + "\"")
-                            .lineLimit(2)
-                            .font(.body)
-                            .foregroundColor(.primaryTxt)
-                            .tint(.accent)
-                            .handleURLsInRouter()
-                        
-                        if viewModel.content == nil {
-                            contentText.redacted(reason: .placeholder)
-                        } else {
-                            contentText
+                    if let contentText = content, !contentText.characters.isEmpty {
```

Comment:
Since we're using `let contentText` below, I'd prefer just:

```suggestion
                    if let content, !content.characters.isEmpty {
```

Reply:
Yep, that's better. Thanks.